### DISCUSSION TITLE
Question how run BenchmarkDotnet

### DISCUSSION THREAD

**Discussion by kronic** [dotnet/runtime#60146] [Q&A]
I am created [pr](https://github.com/dotnet/runtime/pull/60076). Can you suggest how can I run the Test Code of the BenchmarkDotnet?

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one [area label](https://github.com/dotnet/runtime/blob/master/docs/area-owners.md).

**Comment by danmoseley** [comment]
@kronic are you looking for this? https://github.com/dotnet/performance/blob/main/docs/benchmarking-workflow-dotnet-runtime.md

there are other related docs there.
